integration/helpers/base_test.py

```diff
@@ -452,7 +452,7 @@ def verify_stack(self, end_state="CREATE_COMPLETE"):
 
     @retry(
         stop=stop_after_attempt(3),
```

Comment:
should we increase the number of attempts as well?

Reply:
Sure we can do that as well to increase to 5